### PR TITLE
lxc info: show cpu usage

### LINKED ISSUE #1867: CPU use in seconds.
as we know there is a "cpu use" in the output of lxc-info command that show CPU usage of container in seconds, but it does't exist in "lxc info <container>" response. How can I get it? Is there another way of getting CPU clocks that a container used?
Node that lxd version is 2.0.0.rc8

### PR DESCRIPTION
**Pull request by brauner**
Close #1867.

### CODE REVIEW

File: lxc/info.go [hunk 1]
```diff
@@ -141,6 +141,17 @@ func (c *infoCmd) containerInfo(d *lxd.Client, name string, showLog bool) error
 			fmt.Printf(diskInfo)
 		}
 
+		// CPU usage
+		cpuInfo := ""
+		if cs.CPU.Usage != 0 {
+			cpuInfo += fmt.Sprintf("    %s: %v\n", i18n.G("CPU usage (in seconds)"), cs.CPU.Usage / 1000000000)
```
Comment (stgraber) [inline lxc/info.go]: You'll need to run "make i18n" and commit the result, otherwise the testsuite will fail :)

File: lxc/info.go [hunk 2]
```diff
@@ -141,6 +141,17 @@ func (c *infoCmd) containerInfo(d *lxd.Client, name string, showLog bool) error
...
+		}
+
+		if cpuInfo != "" {
+			fmt.Println(i18n.G("  CPU usage:"))
```
Comment (stgraber) [inline lxc/info.go]: That one too (will be included in the same i18n run).

**Comment by stgraber** [comment]
This will need the following:
 - Update to doc/rest-api.md
 - Addition of an API extension (say containter_cpu_time) to the list in lxd/api_1.0.go
 - Update to doc/api-extensions.md for the new API extension

**Comment by brauner** [comment]
Thanks. :)

**Comment by stgraber** [comment]
Needs rebasing apparently.

**Comment by stgraber** [comment]
Looks like you should run "go fmt ./..."

File: lxd/container_lxc.go [hunk 3]
```diff
@@ -3398,6 +3399,25 @@ func (c *containerLXC) Exec(command []string, env map[string]string, stdin *os.F
 	return 0, nil
 }
 
+func (c *containerLXC) cpuState() shared.ContainerStateCPU {
+	cpu := shared.ContainerStateCPU{}
+
+	if !cgCpuController {
+		return cpu
+	}
```
Comment (stgraber) [inline lxd/container_lxc.go]: This checks for the cpu controller, not for cpuacct.

**Comment by brauner** [comment]
"Agent went offline during build." :)

**Comment by stgraber** [comment]
Also known as you managed to kernel panic a test runner :)

**Pull request merged**